Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:8-jdk-alpine
+
+# Set working directory
+WORKDIR /app
+
+# Copy current directory contents into container
+COPY . /app
```

Comment:
this docker file will need to be changed every time we change the version of our project, lets make it more generic. We can also copy only the one file we need.